addons-l10n/en/editor-devtools.json

```diff
@@ -7,16 +7,16 @@
   "editor-devtools/interactive-find-bar": "Interactive Find Bar (Ctrl + F)",
   "editor-devtools/interactive-find-bar-desc": "Quickly find and jump to any Custom Block, Variable, Event, or Hat block defined in a sprite by clicking on the new find bar located to the right of the Code, Costumes and Sound tabs. Begin typing to filter down the list. Use the up and down arrow keys to switch between the possible entries, and the left and right arrows to cycle between all found instances of that block.",
   "editor-devtools/improved-tidy-up": "Improved Code Tidy Up",
-  "editor-devtools/improved-tidy-up-desc": "Right click on the scripts window to pop up the menu and the Clean up Blocks option will have been replaced by a Clean up Blocks + option. Use this to tidy your scripts and it will preserve your scripts columns as well as attempt to align the comments and remove all those orphaned variables, etc.",
+  "editor-devtools/improved-tidy-up-desc": "\"Clean up Blocks\" option on the block context menu is replaced with \"Clean up Blocks +\" option. Use this to tidy your scripts and it will preserve your scripts columns as well as attempt to align the comments and remove all those orphaned variables, etc.",
   "editor-devtools/help-new": "★New★",
   "editor-devtools/help-make-space": "Make Space",
-  "editor-devtools/help-make-space-desc": "Make extra space around a script by right clicking on the code stack and choosing \"Make Space\".",
+  "editor-devtools/help-make-space-desc": "Make extra space around a script choosing \"Make Space\" on the block context menu.",
   "editor-devtools/copy-to-clipboard": "Copy to Clipboard",
-  "editor-devtools/copy-to-clipboard-desc": "Right click a block and 3 new options are available to Copy All, Copy Block, and Cut Block. The Copy All will copy to the clipboard everything including and below the block you clicked on. Copy block will only copy the current block and its contents, but nothing below. And cut block will copy it and remove it from the workspace.",
+  "editor-devtools/copy-to-clipboard-desc": "Block context menu includes 3 new options: Copy All, Copy Block, and Cut Block. The Copy All will copy to the clipboard everything including and below the block you clicked on. Copy block will only copy the current block and its contents, but nothing below. And cut block will copy it and remove it from the workspace.",
```

Comment:
```suggestion
  "editor-devtools/copy-to-clipboard-desc": "Block context menus includes 3 new options: Copy All, Copy Block, and Cut Block. The Copy All will copy to the clipboard everything including and below the block you clicked on. Copy block will only copy the current block and its contents, but nothing below. And cut block will copy it and remove it from the workspace.",
```

Reply:
This is English, we can't just change a word.

Reply:
oh right how bout now